src/bash/editor.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+WD=`dirname $0`
+. ${WD}/include.sh
+PK3="${WD}/CameramanEditor.pk3"
+LOG="${WD}/editor.log"
+ARGS="+freelook 1 +noclip 2 +notarget -file $PK3 +logfile $LOG "
```

Comment:
`+noclip 2` is supposed to be without space (like this `+noclip2`). 
It activates **noclip2** cheatcode, which, unlike regular **noclip**, allows flying.